docs/en/concepts/llms.mdx

```diff
@@ -311,20 +311,47 @@ In this section, you'll find detailed examples that help you select, configure,
   </Accordion>
 
   <Accordion title="AWS Bedrock">
+    Amazon Bedrock supports two authentication methods:
+
+    **Method 1: IAM Role Authentication (Recommended for Production)**
     ```toml Code
     AWS_ACCESS_KEY_ID=<your-access-key>
     AWS_SECRET_ACCESS_KEY=<your-secret-key>
     AWS_DEFAULT_REGION=<your-region>
     ```
 
+    **Method 2: API Key Authentication (Recommended for Development)**
```

Comment:
Perhaps include a link to the Bedrock docs
https://docs.aws.amazon.com/bedrock/latest/userguide/api-keys-use.html